knockoff.gemspec

```diff
@@ -13,13 +13,14 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/sgringwe/knockoff"
   spec.license       = "MIT"
 
+  spec.required_ruby_version = '>= 2.7.0'
   spec.platform      = Gem::Platform::RUBY
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.bindir        = "bin"
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency 'activerecord', '>= 6', '< 7'
-  spec.add_runtime_dependency 'activesupport', '>= 6', '< 7'
+  spec.add_runtime_dependency 'activerecord', '>= 6.1', "< 7.1"
+  spec.add_runtime_dependency 'activesupport', '>= 6.1', "< 7.1"
```

Comment:
Does this version, 1.5.0 work on Rails 6.1? I thought it would only work on 7. 

Also, are we concerned about 7.1 and 7.2? It'd be nice not to revisit this work in a few months.

Reply:
Yes, it works on 6.1 but not 6.0

I am concerned about them, but I can do some testing to see if they require changes.

Reply:
Confirmed that there are more breaking changes in 7.1